### ISSUE TITLE
replace "hetmech" name

### ISSUE THREAD

**Issue by vincerubinetti** [greenelab/hetmech-frontend#52]
It seems like this project being called "hetmech" causes a little confusion, and doesn't quite make sense given the final context of the app on https://het.io/.

We should either come up with a different, more clear name, or just let it have no name. For the latter, it would just be considered a search page on https://het.io/. If we do the latter, we need to scan through the code and tooltip text for mentions of "hetmech" and figure out a way to work around that.

**Comment by dhimmel** [comment]
I think we could call our method "hetnet connectivity search" or "DWPC-based hetnet connectivity search". That would give the website a title like "Hetionet Connectivity Search" or "Hetnet Connectivity Search for Hetionet v1.0".

**Comment by vincerubinetti** [comment]
I like "Hetionet Connectivity Search". Short, succinct, and descriptive to the right level. The other ones are too wordy for a title, but could be a sub-title, like:

## Hetionet Connectivity Search

### DWPC-based hetnet connectivity search of Hetionet v1.0

_[description/definition of DWPC, other intro info]_

**Comment by vincerubinetti** [comment]
closed by #87 !

**Comment by dhimmel** [comment]
We're renaming `hetmech-backend` to `connectivity-search-backend` and `hetmech-frontend` to `connectivity-search-frontend`. See:

+ https://github.com/hetio/het.io/pull/35
+ https://github.com/greenelab/connectivity-search-frontend/pull/154
